examples/src/ref_finance.rs

```diff
@@ -275,7 +278,10 @@ async fn main() -> anyhow::Result<()> {
         .json()?;
 
     println!("Current WNear deposit: {}", wnear_deposit);
-    assert_eq!(wnear_deposit, parse_near!("100 N").to_string());
+    assert_eq!(
+        wnear_deposit,
+        NearToken::from_near(100).as_yoctonear().to_string()
```

Comment:
Instead of parsing `wnear_deposit` as String, parse it as `NearToken`, so then you can just use:

```suggestion
        wnear_deposit,
        NearToken::from_near(100)
```